### ISSUE TITLE
Scrollbar starts at top of screen when opening new chat

### ISSUE THREAD

**Issue by OrdoFlammae** [yakyak/yakyak#1302]
The scrollbar starts at the top of the screen with Hangouts. This means that the program is constantly downloading older messages (because they're at the top), and if I'm not able to get the scrollbar down, this causes some serious annoyance. Shouldn't the scrollbar start at the bottom?

**Comment by HomerSp** [comment]
What version of yakyak are you using? This should be fixed in the latest version, but do let me know if it isn't and I'll have a look.

**Comment by OrdoFlammae** [comment]
I'm using 1.15.12, which I think is the latest version.

**Comment by nlfog** [comment]
I have similar issue, but not only does opening up a new chat cause the scroll bar to go to the top, but when I do finally scroll to the bottom (latest message) and type something and send it, the view doesn't move down to the latest message I sent, and then a few seconds later, the scrollbar shoots right up to the top again.

Help -> About ->   YakYak v1.5.12-beta (latest)
Manjaro Linux AUR build

**Comment by SirBubbles** [comment]
Same issue, latest yakyak-git from the aur in manjaro.

**Comment by HomerSp** [comment]
Hi all, sorry for the delay, I've been busy lately but I am still looking at this issue.

**Comment by SirBubbles** [comment]
It appears to be an issue with the AUR package. I installed the package for arch/manjaro from this site and the problem did not appear. So I'd talk to the maintainer for the AUR package.

**Comment by bssb** [comment]
Can confirm the issue only exists with the AUR package. Not only that, but the AUR package has an installed size of almost 500MB (double that of the Arch package available for download on this site), and the systray icon does not work.

**Comment by OrdoFlammae** [comment]
OK, I'll figure something else out, and see if I can contact the AUR maintainer. Thanks for the help.

**Comment by mike-pento** [comment]
Hello All,
I am the guy who maintains the yakyak-git AUR package. I have been unable to reproduce the issue talked about here, however, I did notice that the electron version being used was 8.0.0  and it looks like it should be at 11.1.1 as of a few months ago.

I updated the package with version 11.1.1 of electron, if someone here who is seeing the problem wouldn't mind testing it again, that would be very helpful.
